src/Features/CSharp/Portable/SplitStringLiteral/InterpolatedStringSplitter.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#nullable disable
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Formatting;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.CSharp.SplitStringLiteral
+{
+    internal sealed class InterpolatedStringSplitter : StringSplitter
```

Comment:
i assume these were basically unchanged, but git is just unhappy because of the dedentation?

Reply:
if you make these nested types inside StringSplitter, it might make the PR ok since git might be able to match these up.

Reply:
Yes, no changes.

Reply:
Actually, a couple of minor things: TrySplit and TryCreate

Reply:
Should be easier to review now - first commit has most of the changes, second removes nesting.